README.rst

```diff
@@ -11,6 +11,21 @@ About
 =====
 This package provides an ``ExplainCollection`` class that allows PyMongo's Collection methods to be explained_
 
+PyMongoExplain greatly simplifies the amount of effort needed to explain commands.
+For example, suppose we wanted to explain the following ``update_one`` ::
```

Comment:
RST is very fickle. You need to remove the space before `::` or else the `:` won't be rendered.